pkg/cli-runtime/testing/validatable_table.go

```diff
@@ -98,5 +117,11 @@ func (tc ValidatableTestCase) Run(t *testing.T) {
 		if !tc.ShouldValidate && len(tc.ExpectFieldErrors) == 0 {
 			t.Error("one of ShouldValidate=true or ExpectFieldErrors is required")
 		}
+
+		if tc.CleanUp != nil {
+			if err := tc.CleanUp(t, ctx); err != nil {
+				t.Errorf("unexpected error in clean up: %v", err)
+			}
```

Comment:
update the above error msg with `t.Errorf("error during cleanup %v", err)`

Reply:
updated